src/main/features/response/views/response-type.njk

```diff
@@ -16,30 +16,23 @@
           <legend class="visually-hidden">{{ t("Do you owe the money claimed?") }}</legend>
           {% if error %}<span class="error-message">{{ t(error) }}</span>{% endif %}
 
-          {{ radioOption(form = form, name = 'type[value]', value = 'OWE_ALL_PAID_ALL',
-            label = t('I’ve already paid'), bold = true,
-            hint = t("You need to submit a defence explaining how much you’ve paid the claimant and when you paid.")
-          )}}
+          <p>{{ t("You can make a counterclaim against the claimant after you submit your response.") }}</p>
```

Comment:
you are missing the needed configuration for the analytics sender to send the data to google analytics
see this PR:
https://github.com/hmcts/cmc-citizen-frontend/pull/58/files